vignettes/questionnaires.Rmd

```diff
@@ -0,0 +1,627 @@
+  48L, "GDS02-Bored Often", "GDS0204", "VISIT 3", 3, "2013-01-12", "NO", 0L,
+  49L, "GDS02-Good Spirits Most of Time", "GDS0205", "VISIT 3", 3, "2013-01-12", "YES", 0L,
+  50L, "GDS02-Afraid of Something Bad Happening", "GDS0206", "VISIT 3", 3, "2013-01-12", "YES", 1L,
+  51L, "GDS02-Feel Happy Most of Time", "GDS0207", "VISIT 3", 3, "2013-01-12", "NO", 1L,
+  52L, "GDS02-Often Feel Helpless", "GDS0208", "VISIT 3", 3, "2013-01-12", "YES", 1L,
+  53L, "GDS02-Prefer to Stay Home", "GDS0209", "VISIT 3", 3, "2013-01-12", "NO", 0L,
+  54L, "GDS02-Memory Problems", "GDS0210", "VISIT 3", 3, "2013-01-12", "NO", 0L,
+  55L, "GDS02-Wonderful to Be Alive", "GDS0211", "VISIT 3", 3, "2013-01-12", "NO", 1L,
+  56L, "GDS02-Feel Worthless", "GDS0212", "VISIT 3", 3, "2013-01-12", "YES", 1L,
+  57L, "GDS02-Feel Full of Energy", "GDS0213", "VISIT 3", 3, "2013-01-12", "YES", 0L,
+  58L, "GDS02-Feel Hopeless", "GDS0214", "VISIT 3", 3, "2013-01-12", "YES", 1L,
+  59L, "GDS02-Most People Better Off Than You", "GDS0215", "VISIT 3", 3, "2013-01-12", "NO", 0L,
+  60L, "GDS02-Satisfied With Life", "GDS0201", "VISIT 4", 4, "2013-02-13", "YES", 0L,
+  61L, "GDS02-Dropped Activities and Interests", "GDS0202", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  62L, "GDS02-Life Is Empty", "GDS0203", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  63L, "GDS02-Bored Often", "GDS0204", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  64L, "GDS02-Good Spirits Most of Time", "GDS0205", "VISIT 4", 4, "2013-02-13", "YES", 0L,
+  65L, "GDS02-Afraid of Something Bad Happening", "GDS0206", "VISIT 4", 4, "2013-02-13", "YES", 1L,
+  66L, "GDS02-Feel Happy Most of Time", "GDS0207", "VISIT 4", 4, "2013-02-13", "NO", 1L,
+  67L, "GDS02-Often Feel Helpless", "GDS0208", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  68L, "GDS02-Prefer to Stay Home", "GDS0209", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  69L, "GDS02-Memory Problems", "GDS0210", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  70L, "GDS02-Wonderful to Be Alive", "GDS0211", "VISIT 4", 4, "2013-02-13", "NO", 1L,
+  71L, "GDS02-Feel Worthless", "GDS0212", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  72L, "GDS02-Feel Full of Energy", "GDS0213", "VISIT 4", 4, "2013-02-13", "YES", 0L,
+  73L, "GDS02-Feel Hopeless", "GDS0214", "VISIT 4", 4, "2013-02-13", "NO", 0L,
+  74L, "GDS02-Most People Better Off Than You", "GDS0215", "VISIT 4", 4, "2013-02-13", "NO", 0L
+) %>%
+  mutate(QSCAT = "GDS SHORT FORM")
+
+qs <- bind_rows(qs_gad7, qs_gdssf) %>%
+  mutate(
+    STUDYID = "STUDYX",
+    USUBJID = "P0001",
+    .before = everything()
+  )
+
+dataset_vignette(qs)
+
+adsl <- tibble::tribble(
+  ~STUDYID, ~USUBJID, ~SITEID, ~ITTFL,    ~TRTP, ~TRTSDT,                      ~DTHCAUS,
+  "STUDYX",  "P0001",     13L,    "Y", "DRUG A", lubridate::ymd("2012-11-16"), NA_character_
+)
+```
+# Original Items
+
+The original items can be handled in the same way like in a [BDS finding ADaM](bds_finding.html). For example:
```

Comment:
Pardon my ignorance, but is Original Items just the answers to the Questionnaire questions?  Could we maybe give a quick blurb on what that means for the unwashed masses (like me)?

Reply:
I have added some text. Does this help?

Reply:
yes tyty